x/ibc/02-client/types/keys.go

```diff
@@ -0,0 +1,67 @@
+package types
+
+import (
+	"fmt"
+)
+
+const (
+	// SubModuleName defines the IBC client name
+	SubModuleName = "clients"
```

Comment:
`clients` or `client`?

Reply:
I had this question as well. `clients` is used for the store paths, but I like `client` better